app/harvesters/cdl_harvester.rb

```diff
@@ -0,0 +1,50 @@
+##
+# A harvester for CDL's Solr API-endpoint
+#
+#
+# @see Krikri::Harvesters::ApiHarvester
+class CdlHarvester < Krikri::Harvesters::ApiHarvester
+ include Krikri::Harvester
+  attr_reader :opts
+
+  ##
+  # @param opts [Hash] options for the harvester
```

Comment:
Can you provide an `@example` annotation about how to invoke the harvester? I'm not entirely clear how to do that, especially since the CDL API requires an authentication token.